### ISSUE TITLE
Request: Android App release on Google Play

### ISSUE THREAD

**Issue by hzulla** [janbar/noson-app#226]
Hey there,

thanks (again) for the wonderful Noson application.

Given the debacle of the new official Sonos app where [architectural issues](https://www.linkedin.com/pulse/what-happened-sonos-app-technical-analysis-andy-pennell-wigwc) seem to make it unlikely that Sonos will be able to fix their app in the medium future, I'd like to request that you publish the Android version of Noson on Google Play.

I'd pay for a working 3rd party Sonos controller app on Android and would want Noson to be that app.

Thank you & keep doing the good things you do.

**Comment by liamdv4** [comment]
Agreed. Sonos' business model worried me from day one and I even wrote to them at one stage saying, "Please introduce a subscription model to support app development and maintenance". They didn't listen to me and decided to put all their chips into voice control, instead :-)
